barretenberg/cpp/src/barretenberg/stdlib/primitives/uint/uint.cpp

```diff
@@ -251,14 +251,6 @@ template <typename Builder, typename Native> uint<Builder, Native> uint<Builder,
 
     if (witness_status == WitnessStatus::NOT_NORMALIZED) {
         weak_normalize();
-        /**
-         * constrain_accumulators will do more for PlookupBuilder, but in TurboPLONK it just imposes
```

Comment:
FYI this code path is not used by Ultra, but it seems fine to delete this code path nonetheless.